### DISCUSSION TITLE
Help and advice

### DISCUSSION THREAD

**Discussion by WithLoveCW** [community/community#55516] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I’m having an issue with my GitHub pages. The web pages I’ve built that were working for months have stopped functioning in the last day, the images on the pages have turned blank with the images not showing

**Comment by parita-yadav-db** [comment]
HI Team 
When i am trying to  push changes giving below error pls help on this
yadapar@INPNQLT612590 MINGW64 ~/agmpv_bitbuckt to git_migration (main)
$ git push -u origin main
fatal: unable to access 'https://github.com/parita-yadav-db/git_agmpv.git/': Received HTTP code 403 from proxy after CONNECT

**Reply by bartvdbraak** [reply]
Hi there! It seems you're encountering an error while trying to push changes to your Git repository. The error message indicates that you received a "403 Forbidden" response from the proxy server when attempting to access the repository URL.

The "403 Forbidden" error typically occurs when there is a problem with authentication or access permissions. To troubleshoot this issue, here are a few suggestions:

1. Verify Repository Access: Double-check that you have the necessary permissions to push changes to the repository. Make sure you are the owner or have the required access rights. You can also confirm this by trying to access the repository directly through your web browser.

2. Proxy Configuration: The error message suggests that there may be a problem with your proxy server. Ensure that your Git client is configured correctly to work with the proxy. You might need to update your proxy settings or check with your network administrator for any restrictions or configuration changes.

3. Authentication Credentials: Ensure that you have the correct username and password or access token configured for your Git client. Double-check that the credentials you're using are valid and have the necessary permissions to access the repository.

4. HTTPS vs. SSH: Consider using SSH instead of HTTPS for repository access. SSH can bypass some proxy-related issues. To set up SSH, you'll need to generate an SSH key pair, add the public key to your GitHub account (or the hosting platform you're using), and update your Git configuration to use SSH URLs for repository access.

Please try these troubleshooting steps and let me know if the issue persists. If you have any further details or error messages, please provide them so that we can assist you more effectively.

To authenticate using HTTP to push changes to a GitHub repository, you can use either your GitHub personal access token. Here's a step-by-step guide for each 

1. Generate a personal access token on GitHub. To do this:
  - Go to your GitHub account settings.
  - Navigate to "Developer settings" or "Developer settings > Personal access tokens".
  - Click on "Generate new token" or a similar option.
  - Provide a token description and select the desired scopes or permissions for the token. For pushing changes, the "repo" scope is typically required.
  - Click on "Generate token" or a similar button to create the token.
2. Copy the generated personal access token to your clipboard.
3. When you run the git push command, instead of entering your GitHub password, paste the personal access token as the password in the Git prompt.

Remember to keep your personal access tokens secure and avoid sharing them publicly. It's generally recommended to use access tokens over passwords for authentication, as tokens can be more easily revoked and managed.

I hope this helps! Let me know if you have any further questions or need more **assistance.**

**Comment by bartvdbraak** [comment, marked as answer]
Hello @WithLoveCW! I noticed that you're facing some issues with your GitHub Pages deployment and the images on your web pages are not showing up. Upon inspecting the console errors, here are a few observations that might help you resolve the problems:

    Mixed Content Block:
        The error message "Mixed Block" suggests that your website is trying to load resources (specifically, wScratchPad.min.js) over both HTTP and HTTPS protocols.
        To fix this, ensure that all your resources, including JavaScript files, CSS files, and images, are being loaded securely using HTTPS. Check if the URLs for your resources start with https:// rather than http://.

    Suspended Page:
        The error "http://jennamolby.com/cgi-sys/suspendedpage.cgi" indicates that the resource at jennamolby.com is suspended, and the JavaScript file wScratchPad.min.js cannot be loaded from that source.
        Verify that the resource hosting the JavaScript file is active and accessible. If it's suspended or unavailable, consider finding an alternative source or hosting the file locally.

    wScratchPad.min.js Function Error:
        The error "Uncaught TypeError: $(...).wScratchPad is not a function" suggests that the function wScratchPad is not being recognized or loaded properly.
        Double-check that you have included the necessary JavaScript file (wScratchPad.min.js) in your project and that it is being loaded before the script that uses the wScratchPad function.
        Ensure that the file path to wScratchPad.min.js is correct and that the file is not missing or corrupted.

    Favicon Not Found:
        The "404 Not Found" error for favicon.ico indicates that the favicon file (favicon.ico) is missing or inaccessible.
        Verify that the favicon file is present in the root directory of your project, named favicon.ico, and that it is accessible.

Consider addressing these issues to resolve the problems you're encountering. If you require further assistance or if there's specific code you'd like me to review, please provide more details, and I'll be happy to help you further. Good luck with fixing the issues on your GitHub Pages deployment!

**Reply by WithLoveCW** [reply]
Hello @bartvdbraak thank you very much for your reply! I’m still very new to the world of html and web code so really appreciate the help 

I suspect the website being suspended is the reason my webpages are no longer working. This was a resource where it helped me create a digital scratch card although I didn’t realise it was pulling from that other website. I basically have two images on the webpage and one sits on top of the other and can be scratched like a scratch card. Hence the scratchpad within my code.

I would be ever so appreciative if you can please help me with the code. I’m not sure how I would host that Java script locally? I have a wix website, not sure if that’s helpful to host the files that need hosting? Once this is resolved will my code work exactly as it did before? 

Kind regards 
Charlotte

**Reply by bartvdbraak** [reply]
I created a pull request with changes to fix your issues, you can see them here: https://github.com/WithLoveCW/GR-Surprise-4729/pull/2

If my reply was useful to you, please consider **marking it as answer** to your question.

**Reply by bartvdbraak** [reply]
See the picture below:

![image](https://github.com/community/community/assets/3996360/c8ff9d7d-1937-4ca2-a636-546460d4d631)

**Reply by bartvdbraak** [reply]
You can see the new changes in your latest Github Pages deployment:
- https://withlovecw.github.io/GR-Surprise-4729/

**Reply by bartvdbraak** [reply]
Hi @WithLoveCW 

As you can see, this discussion thread still is seen as **unanswered**. 

Could you please mark the following as the answer?
https://github.com/orgs/community/discussions/55516#discussioncomment-5910486

**Reply by WithLoveCW** [reply]
@bartvdbraak absolutely thank you for your help!
